akamai/cache-client.go

```diff
@@ -205,8 +205,11 @@ func (cpc *CachePurgeClient) purge(urls []string) error {
 	if err != nil {
 		return err
 	}
-	if purgeInfo.HTTPStatus != 201 || resp.StatusCode != 201 {
-		return fmt.Errorf("Incorrect HTTP status code: %s", string(body))
+	if purgeInfo.HTTPStatus != http.StatusCreated || resp.StatusCode != http.StatusCreated {
+		if purgeInfo.HTTPStatus == http.StatusForbidden {
+			return errFatal(fmt.Sprintf("Unauthorized to purge URLs %q", urls))
+		}
+		return fmt.Errorf("Unexpected HTTP status code: %s", string(body))
```

Comment:
The error message here says "Unexpected HTTP status code" but logs the response body. Should it log `resp.StatusCode` as well? I think the `purgeInfo.HTTPStatus` would be reflected in the body already in its marshalled form, right?